services-geojson/src/main/java/com/mapbox/geojson/CoordinateContainer.java

```diff
@@ -17,5 +17,5 @@
    * @return the {@link Point}s which make up the coordinates defining the geometry
    * @since 3.0.0
    */
-  T coordinates();
+  public abstract T coordinates();
```

Comment:
the previous method was package private, can we retain that with this setup?

Reply:
This was part of an interface so when implemented it would have to be public